docs/generate-evidence-strings.md

```diff
@@ -113,7 +113,13 @@ After the evidence strings have been generated, summary metrics need to be updat
 ### Submit evidence strings
 The evidence string file (`evidence_strings.json`) must be uploaded to the [Open Targets Google Cloud Storage](https://console.cloud.google.com/storage/browser/otar012-eva/) and be named in the format `cttv012-[yyyy]-[mm]-[dd].json.gz` (e.g. `cttv012-2020-10-21.json.gz`).
 
-More details can be found on [Open Targets Github wiki](https://github.com/opentargets/data_release/wiki/OT006-Data-Submission#ot009-evidence-string-generation-json-schema-validation--submission).
+Once the upload is complete, send an email to Open Targets helpdesk containing the following information:
+* The number of submitted evidence strings
+* The ClinVar release date
+* The Ensembl release (depends on the evidence string generation date)
+* The EFO version used for mapping (depends on the manual curation date)
+* The `eva-opentargets` pipeline version
+* The Open Targets JSON schema version
```

Comment:
If possible I would add some **simple instructions or a few commands** to check all these details. That way, whoever is the responsible to do such report it would always produce the same output and thus would be standard (e.g. I may count evidence strings by counting lines, and someone else could do that by counting special characters; I may report dates in a wrong format... etc.). 

Besides, I would also put between brackets in each line **an example** of what is supposed to be reported, so that it is also standardized. For instance:

- The ClinVar release date (e.g. ``ClinVar release date: 2021-06-04``)

Reply:
This is a good point, these were mostly already reported in the metrics spreadsheet, I updated this to actually mention that and added the missing versions and how to determine them.